doc/classes/InputEventKey.xml

```diff
@@ -63,16 +64,16 @@
 	</methods>
 	<members>
 		<member name="echo" type="bool" setter="set_echo" getter="is_echo">
-			Echo state of the key, i.e. whether it's a repeat event or not.
+			[code]true[/code] if the same key was already pressed before this event, means the key is held down.
```

Comment:
`[code]true[/code] if the key was already pressed before this event. It means the user is holding the key down.`